code/datums/components/holderloving.dm

```diff
@@ -0,0 +1,48 @@
+/// Prevents items from leaving their storage or whoever is holding their storage (does not need to have a storage component)
+/datum/component/holderloving
+	can_transfer = TRUE
+	/// Item that is kept track of
+	var/obj/item/holder
```

Comment:
I'm confused. It looks like parent is an item, so then what is the holder if it's *also* an item?

Reply:
This component keeps the parent either directly inside holder, or inside of the holder's loc if that loc is a mob.

This is like defibs, the water tank backpack, etc. In this case the defib is the holder, and the paddles are the parent that the component is put in

Reply:
How would you suggest making this clearer?

Reply:
Putting all this in the doc-comment, because I'm still confused 😄

Reply:
I'm still confused and still think this needs to be explained in a doc comment.

Reply:
Does this help?